home/views.py

```diff
@@ -7,3 +9,37 @@ def about(request):
 
 def landingpage(request):
     return render(request, 'home/landingpage.html')
+
+
+class QuestionDetailView(DetailView):
```

Comment:
Not sure what is the usefulnes of having class-based views here, IMO those seem to introduce extra code builerplate without visible benefits.

Reply:
changed to FBV